Microsoft.DotNet.Interactive.Tests/Language.cs

```diff
@@ -19,7 +19,7 @@ public static string LanguageName(this Language language)
             {
                 Language.CSharp => "csharp",
                 Language.FSharp => "fsharp",
-                Language.PowerShell => "pwsh",
+                Language.PowerShell => "powershell",
```

Comment:
this will alter also magic commands in notebooks samples

Reply:
This is a test-only change, so it won't affect any magic commands.  I found that executing `/variables/pwsh/x` wouldn't find the right kernel, but `/variables/powershell/x` would, so I had to change this because I created it wrong in the beginning.